src/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -459,7 +459,7 @@ export class FlightPlanManager {
             fplnIndex = this._currentFlightPlanIndex;
         }
 
-        const destIndex = this.getDestinationIndex(fplnIndex);
+        const destIndex = this.getDestinationIndex();
```

Comment:
That is really a bug in getDestinationIndex, as it would mean we can't for instance access the active flightplan while the temporary flight plan is being edited, and all of the flight guidance and various FM functions should always access the active flight plan.